src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala

```diff
@@ -173,4 +173,11 @@ class BuiltinStringFunctionsTest extends AnyFlatSpec with Matchers with FeelInte
 
     eval(" string length(uuid()) ") should be(ValNumber(36))
   }
+
+  it should "return a string encoded as base64" in {
+
+    eval(""" encode base64("FEEL") """) should be(ValString("RkVFTA=="))
+
+    eval(""" encode base64("Camunda") """) should be(ValString("Q2FtdW5kYQ=="))
```

Comment:
:wrench: We should add another test case that invokes the function with the named argument. This ensure that we don't change the parameter name accidentally. 

```
eval(""" encode base64(string: "Camunda") """) should be(ValString("Q2FtdW5kYQ=="))
```